### DISCUSSION TITLE
next js 13 app dir image warning

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#44684] [Help]
hellow. I have a warning about height and width. how can solve id.

this is warning in chorme console:

Image with src "/images/middle-banner/banner1.jpg" has either width or height modified, but not the other. If you use CSS to change the size of your image, also include the styles 'width: "auto"' or 'height: "auto"' to maintain the aspect ratio.

thank's for your responses...

**Comment by icyJoseph** [comment]
Could you should how you are rendering the image? With code?

**Reply by ghost** [reply]
this is the code

```
<div className=" flex justify-start items-center ">
    <Image
         className=" rounded-lg"
         src={data.image}
         alt={data.imageAlt}
         title={data.imageAlt}
         width={400}
         height={200}
     />
</div>
```


data comes from props.
